packages/apps/proxy/src/controllers/featuresController.ts

```diff
@@ -192,6 +194,7 @@ export const featuresRouter = (ctx: Context) => {
     "/proxy/features",
     apiKeyMiddleware,
     express.json({
+      limit: process.env.MAX_PAYLOAD_SIZE ?? "2mb",
```

Comment:
nit: should we move the "2mb" out to a const since we use it more than once and if we update the default we only need to update it in one spot

Reply:
good call, fixed